iree/tools/utils/yaml_util.c

```diff
@@ -0,0 +1,16 @@
+// Copyright 2021 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree/tools/utils/yaml_util.h"
+
+#include <yaml.h>
+
+// TODO(benvanik): yaml parsing/printing to vm types.
```

Comment:
What "more meaningful" uses are you planning with this new dep?
Can you elaborate on this TODO?

Reply:
This is to connect to what stella is doing with dumping traces of IREE runtime calls from the python side - python will produce a stream of events in yaml files like "load module" and "call" and this code will parse those to make the C API calls. This will let us model more complex calls (nested lists of lists of buffer views and primitive types, etc) and then produce yaml files with the call results for verification.